.ci/clusters/values-apache-upgrade.yaml

```diff
@@ -0,0 +1,1023 @@
+#
```

Comment:
What do we need a separate `upgrade values file? Can we use the `values-apache-default.yaml` to upgrade?